src/components/loader/loader-square.style.ts

```diff
@@ -8,6 +8,9 @@ export interface StyledLoaderSquareProps {
   isInsideButton?: boolean;
   /** Applies slate color. Available only when isInsideButton is true. */
   isActive?: boolean;
+  /** Toggle between the default variant and gradient variant */
+  color?: "default" | "gradient";
+  loaderNumber?: number;
```

Comment:
comment: If you do keep the same approach here I think it's worth adding a comment even if the new prop isn't external facing